README.md

```diff
@@ -1,6 +1,57 @@
-# LLVM-based Solidity Compiler Toolchain
+<div align="center">
+  <img src=".github/assets/logo.png" alt="solx logo" />
+</div>
 
-This repository contains an LLVM-based Compiler Toolchain for Solidity and Yul.
+### `solx` is an experimental compiler for Solidity
+
+The goal of the project is to make state-of-the-art optimizing compilers a mainstream for Web3.

```

Comment:
The project aims to bring state-of-the-art optimizing compilers to Ethereum by:

* Reducing gas consumption of EVM code.
* Eliminating the need for inline assembly.
* Providing industry-grade tools like an interactive debugger.

but overall too many promises and lack of current state.
Probably it's worth mentioning the tests solx passes and highlight benchmark results.